src/components/Filters/README.md

```diff
@@ -0,0 +1,483 @@
+
+---
+
+## Accessibility
+
+The filters component relies on the accessibility features of multiple other components:
+
+- [Text field](https://polaris.shopify.com/components/forms/text-field)
+- [Button](https://polaris.shopify.com/components/actions/button)
+- [Popover](https://polaris.shopify.com/components/overlays/popover)
+- [Sheet](https://polaris.shopify.com/components/overlays/sheet)
+- [Collapsible](https://polaris.shopify.com/components/behavior/collapsible)
+
+### Maintain accessibility with custom features
+
+Since custom HTML can be passed to the component for additional actions, ensure that the filtering system you build is accessible as a whole.
+
+All merchants must:
+
+- be able to identify and understand labels for all controls
+- be notified of state changes
+- be able to complete all actions with the keyboard
+
+---
+
+## Best practices
+
+The filters component should:
+
+- help reduce merchant effort by promoting the filtering categories that are most commonly used
+- include no more than 2 or 3 promoted filters
+- consider small screen sizes when deciding the interface for each filter and the total number filters to include
+- let merchants select more than one filter from within the same category. For example, in the payment filter category, merchants can combine paid, partially paid, and unpaid
+- use the filters action only for an action that’s related or relevant to filtering
+
+---
+
+## Content guidelines
+
+### Text field
+
+The text field should be clearly labeled so it’s obvious to merchants what they should enter into the field.
+
+<!-- usagelist -->
+
+#### Do
+
+- Filter orders
+
+#### Don’t
+
+- Enter text here
+
+<!-- end -->
+
+### Filter badges
+
+Use the name of the filter if the purpose of the name is clear on its own. For example, when you see a filter badge that reads **Paid**, it’s intuitive that it falls under the Payment status category.
+
+<!-- usagelist -->
+
+#### Do
+
+- Paid, Unpaid
+
+#### Don’t
+
+- Order status: Paid, Unpaid
+
+<!-- end -->
+
+If the filter name is ambiguous on its own, add a descriptive word related to the status. For example, **Low** doesn’t make sense out of context. Add the word “risk” so that merchants know it’s from the Risk category.
+
+<!-- usagelist -->
+
+#### Do
+
+- High risk, Low risk
+
+#### Don’t
+
+- High, Low
+
+<!-- end -->
+
+Group tags from the same category together.
+
+<!-- usagelist -->
+
+#### Do
+
+- (Unfulfilled, Fulfilled)
+
+#### Don’t
+
+- (Unfulfilled) (fulfilled)
+
+<!-- end -->
+
+If all tag pills selected: truncate in the middle
```

Comment:
We might want to remove this guideline, since this doesn't appear to be what we're doing with order filters? 

Live in the product:
![alt](https://screenshot.click/26-57-vy78z-49b3u.jpg)

Reply:
"Do as I say, not as I do" 😆

Reply:
Both of these are concerns with the badge and tag component. We would need to make this change in tag and then for badge it would be up to the consumer (maybe?). Not great for us to use it differently than we recommend, but also think that is okay.